package.json

```diff
@@ -13,6 +13,7 @@
     "axios": "^0.21.4",
     "bitcore-lib": "^8.25.10",
     "bitcore-mnemonic": "^8.25.10",
+    "buffer": "^6.0.3",
```

Comment:
I think it would be good to add just a short explanation to this PR's description on why did we add the `buffer` lib now, instead of relying on the runtime's native one.

Reply:
The buffer was already present on our lib but it was a dependency of some other package.
The issue is that since we actually import it on our lib we should have it explicitly on the package.json file.

It may be important later as a polyfill if needed but currently we use it as a dependency.